rest/swagger.json

```
@@ -771,6 +771,16 @@
                 "parameters": [
                     {
                         "type": "string",
+                        "x-go-type": {
+                          "type": "UUID",
+                          "import": {
+                            "package": "github.com/SKF/go-utility/v2/uuid"
+                          },
+                          "hints": {
+                            "kind": "object"
+                          }
+                        },
+                        "x-go-type": { "type": "UUID", "import": { "package": "github.com/SKF/go-utility/v2/uuid" }, "hints": { "kind": "object" } },
```

Some of the schemas contains double definitions of the `x-go-type`
```suggestion
                        "x-go-type": {
                          "type": "UUID",
                          "import": {
                            "package": "github.com/SKF/go-utility/v2/uuid"
                          },
                          "hints": {
                            "kind": "object"
                          }
                        },
```

---

SED once never twice, fixed